src/utils.ts

```diff
@@ -8,7 +8,9 @@ import filterObject = require('filter-object');
 export type SelectedRefinement = SelectedValueRefinement & SelectedRangeRefinement;
 
 export function findSearchBox() {
-  return <HTMLInputElement>oget(findTag('gb-query'), '_tag.searchBox');
+  if (findTag('gb-search-box') && findTag('gb-search-box').getElementsByTagName('input')) {
+    return <HTMLInputElement>findTag('gb-search-box').getElementsByTagName('input')[0];
```

Comment:
prefer the original approach so that we can pick up input fields that aren't tagged as `gb-search-box`. To test, should be able to stub with
```
import * as utils from '../../src/utils';

sandbox.stub(utils, 'findTag', () => <some tag>);
```